src/features/tasks/tasksSlice.js

```diff
@@ -505,6 +624,67 @@ export const updateCameraSerialNumber = (payload) => {
   };
 };
 
+// delete images thunk
+export const deleteImagesTask = ({ imageIds }) => {
+  return async (dispatch, getState) => {
+    try {
+      dispatch(deleteImagesStart());
+      const currentUser = await Auth.currentAuthenticatedUser();
+      const token = currentUser.getSignInUserSession().getIdToken().getJwtToken();
+      const projects = getState().projects.projects;
+      const selectedProj = projects.find((proj) => proj.selected);
+
+      if (token && selectedProj) {
+        const res = await call({
+          projId: selectedProj._id,
+          request: 'deleteImagesTask',
+          input: { imageIds },
+        });
+        console.log('deleteImages - res: ', res);
+        dispatch(deleteImagesUpdate({ taskId: res.deleteImagesTask._id }));
+        dispatch(
+          setFocus({
+            index: { image: null, object: null, label: null },
+            type: 'auto',
+          }),
+        );
+        dispatch(setSelectedImageIndices([]));
+        dispatch(setDeleteImagesAlertOpen(false));
+      }
+    }
+    catch (err) {
+      dispatch(deleteImagesFailure(err));
+    }
+  };
+};
+
+// delete images by filter thunk
+export const deleteImagesByFilterTask = ({ filters }) => {
+  return async (dispatch, getState) => {
+    try {
+      dispatch(deleteImagesByFilterStart());
+      const currentUser = await Auth.currentAuthenticatedUser();
+      const token = currentUser.getSignInUserSession().getIdToken().getJwtToken();
+      const projects = getState().projects.projects;
+      const selectedProj = projects.find((proj) => proj.selected);
+      console.log(filters);
+
+      if (token && selectedProj) {
+        const res = await call({
+          projId: selectedProj._id,
+          request: 'deleteImagesByFilterTask',
+          input: { filters },
+        });
+        console.log('deleteImagesByFilter - res: ', res);
+        dispatch(deleteImagesByFilterUpdate({ taskId: res.deleteImagesByFilterTask._id }));
```

Comment:
Correct me if I'm wrong but I think we might want to be clearing the focus and selected indices just like we do in `deleteImagesTask` thunk above on line 645 just in case someone has an image (or multiple images) selected when they initiate the delete-by-filter task.